terraform/README.md

```diff
@@ -3,11 +3,22 @@
 ## Directory structure
 
 * `deployments`: root modules, from where you can run Terraform commands. These
-  should only call the `govuk` composition module and nothing else.
+  should only call the `govuk` composition module, and individual app deployment
+  modules.
+    * `govuk-test`: calls the `govuk` composition module to bring up the core
+      infrastructure
```

Comment:
I think we should replace the phrase "core infrastructure" with something more specific. Would it be accurate enough to say that it brings up everything except for the task definitions?